src/main/java/calculator/Operator.java

```diff
@@ -0,0 +1,52 @@
+package calculator;
+
+import java.util.Arrays;
+
+import static calculator.NumberParser.checkMathOperator;
+
+public enum Operator {
+
+    ADDITION("+") {
+        long calculate(long x, long y) {
+            return x + y;
+        }
```

Comment:
```suggestion
    ADDITION("+", (x, y) -> x + y)
```

람다 표현식을 적극적으로 활용해보는 것은 어떨까요